core/src/test/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityTest.java

```diff
@@ -71,7 +71,28 @@
       + " {\"type\":\"string\",\"name\":\"f2\", \"default\": \"foo\"}]},"
       + " {\"type\":\"string\",\"name\":\"f3\", \"default\": \"bar\"}]}";
   private final Schema schema8 = AvroUtils.parseSchema(schemaString8).schemaObj;
-  
+
+  private final String schemaString9 = "{\"type\":\"record\","
```

Comment:
Is this part of your change?

Reply:
I added these schema for my new test cases